### PR TITLE
checkbox component was added

### PR DESCRIPTION
**Pull request by Exper1ment4L**

### CODE REVIEW

File: src/components/Checkbox/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    ${HiddenCheckbox}:checked + & {
+        background: #18a0fb;
+        border-color: #18a0fb;
+    }
+    &:hover {
+        border-color: #18a0fb;
+    }
+`;
+
+const Checkbox = ({ checked, icon, ...props }) => (
```
Comment (aslanyi) [inline src/components/Checkbox/index.js:34]: Checkbox should has label. User can click the label to check.

File: src/components/Checkbox/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+`;
+
+const StyledCheckbox = styled.div`
+    width: 15px;
+    height: 15px;
+    display: inline-block;
+    border: 2px solid #d5d5d5;
+    border-radius: 4px;
+    ${HiddenCheckbox}:checked + & {
+        background: #18a0fb;
```
Comment (aslanyi) [inline src/components/Checkbox/index.js:26]: We should use theme for colors.

**Comment by aslanyi** [comment]
Screen Shot 2020-05-31 at 23 22 28

This is not a good view for user experience.

File: src/components/Checkbox/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import styled from 'styled-components';
+import propTypes from 'prop-types';
+import Icon from '../../utils/icon';
+
+const CheckboxContainer = styled.div`
+    position: relative;
+`;
+
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
+    width: 1px;
```
[the remark this answers is not in the document]
Comment (Exper1ment4L) [reply src/components/Checkbox/index.js]: We have not that properties. That was in outdated file.

**Pull request merged**